mkdocs.yml

```diff
@@ -0,0 +1,70 @@
+site_name: "Diffusers-Interpret"
+site_description: "Model explainer tool built on top of the 🤗 Diffusers library"
+repo_name: "JoaoLages/diffusers-interpret"
+repo_url: "https://github.com/JoaoLages/diffusers-interpret"
+
+theme:
+  name: "material"
+  features:
+    - navigation.top
+    - navigation.tabs
+  palette:
+  # Palette toggle for automatic mode
+  - media: "(prefers-color-scheme)"
+    primary: yellow
+    accent: red
+    toggle:
+      icon: material/brightness-auto
+      name: Switch to light mode
```

Comment:
The [automatic theme mode] appears to be sponsors-only. Maybe I should comment it out.